12-watchtower-API.md

```diff
@@ -127,122 +129,167 @@ This message contains all the information regarding the appointment that the cli
 
-- ```start_block```: (Absolute) Block number 
-- ```end_block```: (Absolute) Block number
-- ```dispute_delta```: (Relative) Block number
-- ```transaction_size```: Measured in Bytes (e.g. 200) 
-- ```transaction_fee```: Measured in sats (e.g. 2000)
-- ```cipher```: AESGCM256, CHACHA20
-- ```customer_signature_algorithm```: ECDSA, SCHNORR
+The _explictiness_ of the signed job ensures there is a clear protocol transcript between the customer and WatchTower. Given the blockchain and decrypted justice transaction, anyone can verify that the WatchTower could have satisfied the job. 
 
-### Rationale
+### The `appointment_accepted` message
 
-We can group the data fields into logical groups. 
+This message contains information about the acceptance of an appointment from the WatchTower.
 
-* **Appointment information**: The appointment time is defined using the ```start_block``` and```end_block```. WatchTower will delete the job when the appointment has expired. We recommend only using block numbers as that is the natural clock for Bitcoin. 
-* **Explicit acknowledgement of transaction details**: The ```dispute_delta```, ```transaction_size```, and ```transaction_fee``` let the WatchTower confirm the transaction is "reasonable" and it can be accepted into the blockchain (especially if there is congestion in the future). 
-* **Encrypted transaction**: The ```cipher```,```encrypted_blob``` states how the WatchTower can later find the dispute transaction and decrypt the justice transaction. 
-* **Customer signature** The ```customer_public_key``` and ```customer_signature``` provides an explicit message about the job from the customer. 
+1. type: ? (`appointment_accepted `)
+2. data:
+   * [`16*byte `:`locator`]
+   * [`varsize`: `op_receipt`]
+   * [`u16`: `op_wt_signature_algorithm`]
+   * [`varsize`: `op_wt_signature`]
+   * [`varsize`: `op_wt_public_key`]
 
-Generally, this standard is trying to allow a reputationally accountable watching service. The signed job from the customer provides an explicit acknowledgement of the transaction details that is important for the WatchTower to decide whether they can accept it. If the decrypted justice transaction does not satisfy the signed job (e.g. fee too low), then the WatchTower is not obliged to fulfil it. 
+The sending node:
 
-The _explictiness_ of the signed job ensures there is a clear protocol trasncript between the customer and WatchTower. Given the blockchain and decrypted justice transaction, anyone can verify that the WatchTower could have satisified the job. 
+* MUST receive `appointment` before sending an `appointment_accepted` message.
+* MUST set the `locator` to match the one received in `appointment`.
+* if `qos` was agreed on `wt_init`:
+	* MUST set `op_receipt`] to a receipt build according to 	 [Signed-Receipt](#signed-receipt)
```

Comment:
should we call this "Watchtower's response to the user?" instead of sending node? Might be better to add context.

Reply:
It follows the exact same format that the other BOLTS (sending/receiving node). It's a fairly simple protocol so I don't see the problem with it.

Reply:
well issue is that both nodes are sending and receiving. so they take on both roles, so need to look at context to better understand what is going on.

Reply:
I mean it's called `appointment_accepted`. Is an accept message as a response from an user request, I don't think there is any possible misinterpretation